README.md

```diff
@@ -60,229 +35,230 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+- I would like to express my gratitude to the following individuals and resources who have inspired and contributed to this project:
 
-I would like to thank...
+  - [SanaTameem](https://www.linkedin.com/in/sana-tameem-a63b55240/)
+  - [Nemwel Boniface](https://twitter.com/nemwel_bonie)
+
+- Their work and insights have helped me to develop my skills and understanding, and have contributed to the success of this project. I also want to thank the Microverse community for their collective efforts in making software development accessible and collaborative.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
+[Question_1]
 
-- **[Question_1]**
+- How can I customize the visual design of the Rock music concert project to fit my specific needs?
 
   - [Answer_1]
 
-- **[Question_2]**
+    - The visual design of the Rock music concert project can be customized by modifying the CSS code. The project utilizes a combination of flexbox, grid, and media queries to create a responsive layout, and semantic HTML tags to provide structure and accessibility. By tweaking the values of CSS properties such as colors, fonts, and layout rules, you can adjust the look and feel of the project to match your preferences. Additionally, you can add or remove content within the HTML and JavaScript files to tailor the project to your specific use case.
+
+[Question_2]
+
+- How can I add new functionality to the Rock music concert project, such as a feature to purchase tickets?
 
   - [Answer_2]
 
+    - The Rock music concert project is built using JavaScript, which allows for a wide range of custom functionality to be added. To add a feature such as ticket purchasing, you can start by modifying the JavaScript code to create a new function or class that handles the logic of the feature. You may also need to modify the HTML and CSS to add new UI elements and styles that support the new feature. Additionally, you may want to consider integrating third-party libraries or APIs that provide additional functionality, such as payment processing or event scheduling.
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] KIndly restore the MIT license to your readme file. This readme is licensed by MIT so it is required to be in the license section.